storage/innobase/fil/fil0fil.cc

```diff
@@ -8863,8 +8451,8 @@ static dberr_t fil_iterate(const Fil_page_iterator &iter, buf_block_t *block,
       ut_d(block->page.zip.m_external = true);
       ut_ad(iter.m_page_size == callback.get_page_size().physical());
 
-      read_request.mark_page_zip_compressed();
-      read_request.set_zip_page_physical_size(iter.m_page_size);
+      //read_request.mark_page_zip_compressed();
+      //TODO read_request.set_zip_page_physical_size(iter.m_page_size);
```

Comment:
Same here

Reply:
Same, this was related to keyring encryption.